routers/user/setting/security_twofa.go

```diff
@@ -74,11 +74,13 @@ func twofaGenerateSecretAndQr(ctx *context.Context) bool {
 	if uri != nil {
 		otpKey, err = otp.NewKeyFromURL(uri.(string))
 	}
+	// Filter unsafe character ':' in issuer
+	issuer := strings.ReplaceAll(setting.AppName+" ("+setting.Domain+")", ":", "")
```

Comment:
```suggestion
	issuer := strings.Replace(setting.AppName+" ("+setting.Domain+")", ":", "", -1)
```

`strings.ReplaceAll` is new to go1.12  we need to be able to build on go1.10